src/AppBundle/Resources/views/Report/Report/_subsection.html.twig

```diff
@@ -23,33 +26,28 @@
         {% endif %}
     </span>
 
-    {% if allowEdit == true %}
-        <a class="edit-link behat-link-edit-{{ subSection }}" id="edit-{{ subSection }}" href="{{ linkToSubSection }}" aria-label="{{ (subSection ~ '.edit') | trans }}">
-            {% if subSection == 'balance' %}
-                {{ 'view' | trans({}, 'common') | capitalize }}
-            {% else %}
-                {% if state.state == 'not-started' %}
-                    {{ 'start' | trans({}, 'common') | capitalize }}
-                {% else %}
-                    {{ 'edit' | trans({}, 'common') | capitalize }}
-                {% endif %}
-            {% endif %}
-        </a>
+    <span  class="edit-link {% if allowEdit == false %}disabled{% endif %}" {% if allowEdit == true %}href="{{ linkToSubSection }}"{% endif %} >
```

Comment:
here ^^

Reply:
This is consistent with the revert - every version of this file had a href in the <span> tag. I don't agree with it either, but I was just reverting

Reply:
Discussed and this line is reverting the file back to how it was